docs/content/tutorial/advanced-tutorial/repositories.mdx

```diff
@@ -60,17 +42,17 @@ Now you can see the list of all jobs in the repo on the left:
 <Image
 alt="repos.png"
 src="/images/tutorial/repos.png"
-width={4018}
-height={3062}
+width={3808}
+height={2414}
 />
 
 ## Workspaces
 
-Loading repositories via the `-f` or `-m` options is actually just a convenience function. The underlying abstraction is the "workspace", which determines all of the available repositories available to Dagit.
+Loading repositories via the `-f` option is actually just a convenience function. The underlying abstraction is the "workspace", which determines the set of repositories available to Dagit.
```

Comment:
Might be outside of the scope of the changes here, but part of the point of a workspace file is that it allows you to load multiple repository locations at once, right? Maybe worth mentioning.